### ISSUE TITLE
Prefer `t.true(x)` over `t.is(x, true)`

### ISSUE THREAD

**Issue by jonahsnider** [avajs/eslint-plugin-ava#338]
Inspired by https://github.com/sindresorhus/ts-extras/pull/8#discussion_r740023376

Bad:

```js
t.is(x, true);
t.is(x, false);
```

Good:

```js
t.true(x);
t.false(x);
```

**Comment by sindresorhus** [comment]
This should be fixed in the https://github.com/avajs/eslint-plugin-ava/blob/main/docs/rules/use-true-false.md rule.
